test/box-luatest/lua_memory_test.lua

```diff
@@ -0,0 +1,68 @@
+local server = require('luatest.server')
+local t = require('luatest')
+local g = t.group('lua_memory setting test')
+--local treegen = require('test.treegen')
```

Comment:
Looks like a part of the previous version.

Reply:
My bad. Thank you. Fixed!